en/docs/fhir/guides/fhir-repository-connector.md

```diff
@@ -0,0 +1,1082 @@
+# Introduction
+
+Fast Healthcare Interoperability Resources (FHIR) is an interoperability standard for electronic exchange of healthcare information. The WSO2 FHIR Repository connector can be used to seamlessly integrate with a FHIR repository of your choice.
+
```

Comment:
Need to have note on two implementation flavours

Reply:
Added